srcpkgs/gek/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'gek'
+pkgname=gek
+version=0.1
+revision=1
+hostmakedepends="git go"
+short_desc="Stack-based stream editor and scripting language"
+maintainer="John Gebbie <me@johngebbie.com>"
+license="GPL-3.0-only"
+homepage="https://geklang.org/"
+
+do_fetch() {
+	git clone https://git.sr.ht/~geb/gek "${wrksrc}"
```

Comment:
git packages are not accepted. the software must have a release

Reply:
I've got it using --branch v0.1 now, I hope that's good.

Reply:
you should be using `distfiles`, not `do_fetch()`, i.e.: `distfiles="https://git.sr.ht/~geb/gek/archive/v${version}.tar.gz"`

Reply:
bum. I'm not sure how to do that. Do I put archives in the repo manually?  I copied the --branch approach having greped the other templates.

Reply:
sr.ht should auto-generate a tarball when you create a tag (https://git.sr.ht/~geb/gek/archive/v0.1.tar.gz should work)

those `git clone`s in other templates are either very old or special cases

Reply:
Delete do_fetch and let xbps-src use the default one. The default do_fetch function downloads the archive in $distfiles. Look at almost every single template, almost no other template uses git. For example the `vale` package template will have a similar structure.

Reply:
Thank you. I think I've done it.